lib/modules/datasource/github-runners/index.spec.ts

```diff
@@ -3,18 +3,58 @@ import { GithubRunnersDatasource } from '.';
 
 describe('modules/datasource/github-runners/index', () => {
   describe('getReleases', () => {
-    it('returns releases if package is known', async () => {
+    it('returns releases for ubuntu', async () => {
```

Comment:
```suggestion
    it('returns releases for Ubuntu', async () => {
```

We should capitalize proper nouns. 😉